bundles/config/org.eclipse.smarthome.config.xml/config-description-1.0.0.xsd

```diff
@@ -66,6 +67,7 @@
 		<xs:attribute name="required" type="xs:boolean" />
 		<xs:attribute name="readOnly" type="xs:boolean" />
 		<xs:attribute name="multiple" type="xs:boolean" />
+		<xs:attribute name="unit" type="config-description:unitType" />
```

Comment:
I thought taht we agreed only on using a String representation for the Unit and not using a complex unit data type. Because in the ConfigDescriptionParameter constructor the given unit is validated I thought it would be convenient for the developer if an error is highlighted as early as possible. 

Do you prefer if i remove the enum?